src/Controller/ExportController.php

```diff
@@ -56,14 +56,19 @@ public function initialize(): void
     public function export(): ?Response
     {
         $format = (string)$this->request->getData('format');
-        if (!$this->Export->checkFormat($format)) {
-            return null;
-        }
+
         // check request (allowed methods and required parameters)
         $data = $this->checkRequest([
             'allowedMethods' => ['post'],
             'requiredParameters' => ['objectType'],
         ]);
+
+        if (!$this->Export->checkFormat($format)) {
+            $this->Flash->error('Il formato selezionato non è disponibile!');
```

Comment:
We use english for labels and messages with `__()` function to enable translations via gettext
It should be:

```php
$this->Flash->error(_('Selected format....'));
```